Opserver.Core/Data/Dashboard/NodeService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace StackExchange.Opserver.Data.Dashboard
+{
+    /// <summary>
+    /// Service class 
+    /// </summary>
+    public partial class NodeService : IMonitorStatus
+    {
+        public Node Node { get; set; }
+
+        public string Id { get; internal set; }
+        public string NodeId { get; internal set; }
```

Comment:
I don't think we need a `NodeId` here, since the `Node` would always be populated from a can-we-get-to-this standpoint.